GitVersion.yml

```diff
@@ -0,0 +1,3 @@
+next-version: 0.5.2
```

Comment:
This is used as a base and it will look in the history to resolve the correct version. If you decide to use tags at some point, those will take priority, and next-version doesn't get used I believe. At least in Gw2Sharp I don't have this defined and the tags are used instead.